core/geotime/src/main/java/mil/nga/giat/geowave/core/geotime/ingest/SpatialTemporalDimensionalityTypeProvider.java

```diff
@@ -36,7 +36,7 @@ public SpatialTemporalDimensionalityTypeProvider() {}
 
 	@Override
 	public String getDimensionalityTypeName() {
-		return "spatial-temporal";
+		return "spatial_temporal";
```

Comment:
these "dimensionality type names" are now being used in the accumulo table names, before it would pull the index ID built from CustomIdIndex constructors.  The hyphen is an illegal character for accumulo table names and this was the quickest workaround.